src/accelerate/accelerator.py

```diff
@@ -149,6 +150,12 @@
 
 logger = get_logger(__name__)
 
+# Sentinel values for defaults
+_split_batches = object()
+_dispatch_batches = object()
+_even_batches = object()
+_use_seedable_sampler = object()
```

Comment:
Ah, one disadvantage of this is that the type annotation is not correct, which might lead to some issues with IDEs, not sure. But since it's only transitory, I guess we can live with that.

Reply:
They also have the type-hint in the signature which should help.